test/extensions/filters/network/thrift_proxy/router_test.cc

```diff
@@ -355,6 +369,37 @@ INSTANTIATE_TEST_SUITE_P(ContainerFieldTypes, ThriftRouterContainerTest,
                          Values(FieldType::Map, FieldType::List, FieldType::Set),
                          fieldTypeParamToString);
 
+class ThriftRouterPassthroughTest
+    : public testing::TestWithParam<
+          std::tuple<TransportType, ProtocolType, TransportType, ProtocolType>>,
+      public ThriftRouterTestBase {
+public:
```

Comment:
nit: drop this label?

Reply:
It's copied from existing code, and it's used in many places, not sure if there is any particular for keeping this label.